src/phoenix/utilities/logging.py

```diff
@@ -0,0 +1,5 @@
+# A collection of printing and logging utilities
+
+def printif(condition: bool, *args, **kwargs):
+    if condition:
+        print(*args, **kwargs)
```

Comment:
@mikeldking What are your thoughts on the eventual packaging strategy for evals and other Phoenix sub-modules such as our tracers? Are we going to deploy them as distinct packages, e.g., `arize-evals` or `phoenix-evals`? If so, we should be careful about introducing dependencies between the sub-modules and the rest of the codebase.

@anticorrelator  This is a non-blocking comment. We can always move things if needed.

Reply:
Yeah it ideally doesn't sit in phoenix long term so treating it more as a sub-module could be a benefit. I think the verbose logging ask could be evals specific so it could make more sense sitting under evals, though I think this is a trivial change if we do split it so not concerned either way